backend/src/LibExecutionStdLib/LibChar.fs

```diff
@@ -143,4 +143,20 @@ let fns : List<BuiltInFn> =
         | _ -> incorrectArgs ()
       sqlSpec = NotYetImplemented
       previewable = Pure
-      deprecated = NotDeprecated } ]
+      deprecated = NotDeprecated }
+
+
+    { name = fn "Char" "toString" 0
+      parameters = [ Param.make "c" TInt "" ]
+      returnType = TStr
+      description = "Stringify <param c>"
+      fn =
+        (function
+        | _, [ DChar c ] -> Ply(DStr(c))
+        | _ -> incorrectArgs ())
+      sqlSpec = NotQueryable
+      previewable = Pure
+      deprecated = NotDeprecated }
+
+
+    ]
```

Comment:
Is this intended to be a new pattern of ending these fn lists with 2 newlines? (asking as a few files have this)

Reply:
I'm sorta undecided here. It's annoying having the list end at the last entry, but ugly to add space. If you have a preference, happy to go with it.

Reply:
I've been fine with the `} ]` endings. I've played with both locally, and find the empty lines between the two sort of ugly, not worth the tiny diff annoyance when adding a stdlib fn 'at the end'